Editor/PropertyDrawer/MaxDrawer.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEditor;
+using UnityEngine;
+
+[CustomPropertyDrawer( typeof( MaxAttribute ) )]
+public class MaxDrawer : PropertyDrawer
```

Comment:
classes should be in the fishingcactus namespace

Reply:
Done.